shared/test/test_net_sim_api.rb

```diff
@@ -31,7 +32,7 @@ def setup
   def test_create_read_update_delete
     # Verify that the CREATE response body and READ response bodies
     # both return the correct record values
-    record_create_response = create_record({name: 'alice', age: 7, male: false})
+    record_create_response = create_record({type: NODE_TYPES[:client], name: 'alice', age: 7, male: false})
```

Comment:
This indicates to me that we're at a point where we need to start caring a little more about our test data, or at the very least being a bit more explicit about the fact that these records are getting created on the nodes table and therefore have certain requirements.

That, or creating them on another table.